scripts/add_nodes_to_haamek_davar.py

```diff
@@ -0,0 +1,30 @@
+__author__ = 'stevenkaplan'
+from sefaria.helper.schema import *
+torah_titles = library.get_indexes_in_category("Torah")
+he_torah_titles = map(lambda x: library.get_index(x).get_title("he"), torah_titles)
+haamek_titles = map(lambda x: "Haamek Davar on {}".format(x), torah_titles)
+he_haamek_intros = map(lambda x: u"פתיחה לספר {}".format(x), he_torah_titles)
+en_haamek_intros = map(lambda x: "Introduction to {}".format(x), torah_titles)
+
```

Comment:
The 4 lines above could be done with comprehensions.  It's more standard and easier to read. 
```
he_torah_titles = [library.get_index(x).get_title("he") for x in torah_titles]
haamek_titles = ["Haamek Davar on {}".format(x)  for x in torah_titles]
he_haamek_intros = [u"פתיחה לספר {}".format(x) for x in he_torah_titles]
en_haamek_intros = ["Introduction to {}".format(x)  for x in  torah_titles]
```